Scheduler.py

```diff
@@ -0,0 +1,29 @@
+import os
+import UI
+
+from datetime import datetime
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+from apscheduler.triggers.cron import CronTrigger
```

Comment:
where are we using this? is this import necessary?

Reply:
No, it was made for the old constructor but now we dont need it anymore

Reply:
Removed